hackathon_site/event/jinja2/event/landing.html

```diff
@@ -250,18 +250,18 @@ <h3 class="center white-text">FAQ</h3>
                 <!-- Bronze -->
 
                 <!-- Partners offset-m3 offset-s1-->
-<!--                <a href="https://www.ecobee.com/en-ca/" class="col m2 s3 offset-s1" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/ecobee_white.svg') }}" alt="ecobee" /></a>-->
+                <a href="https://mlh.io/" class="col m2 s3 offset-s1" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/mlh-logo-color.svg') }}" alt="ecobee" /></a>-->
```

Comment:
Didn't change the alt text

Reply:
If you remove `offset-s1`, I think it will fix the issue of the thing not being center?